sdks/builds/llvm.mk

```diff
@@ -32,9 +32,18 @@ _llvm_$(1)_CONFIGURE_FLAGS= \
 	--enable-targets="arm,aarch64,x86" \
 	$$(if $$(filter $$(UNAME),Darwin),--enable-libcpp)
 
+_llvm_$(1)_CMAKE_FLAGS = \
+	-DCMAKE_INSTALL_PREFIX=$$(TOP)/sdks/out/llvm-$(1) \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DLLVM_TARGETS_TO_BUILD="X86;ARM;AArch64" \
+	-G Ninja \
```

Comment:
As for Ninja, we don't want to add this forced dependency, because we have no idea if it's supported everywhere we need it to be. Could you replace that by `$$(if $$(shell which ninja),-G ninja)` to make it conditional?